SDKBuildScripts/unity_buildAppCenterTestIOS.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+#USAGE: unity_buildAppCenterTestIOS.sh 
```

Comment:
Oo should we standardize this across scripts? This looks like it would be useful